iidm/iidm-api/src/main/java/com/powsybl/iidm/network/ShuntCompensator.java

```diff
@@ -83,4 +83,77 @@ default double getMaximumB() {
      */
     double getCurrentB();
 
+    /**
+     * Get the terminal used for regulation.
+     */
+    default Terminal getRegulatingTerminal() {
+        throw new UnsupportedOperationException();
+    }
+
+    /**
+     * Set the terminal used for regulation.
+     */
+    default ShuntCompensator setRegulatingTerminal(Terminal regulatingTerminal) {
+        throw new UnsupportedOperationException();
+    }
+
+    /**
+     * Get the shunt compensator's regulating status.
+     * <p>
+     * Depends on the working variant.
+     * @see VariantManager
+     */
+    default boolean isRegulating() {
```

Comment:
Just a question: regulations in IIDM have a different modeling depending on the object. Here you have chosen ````Regulating```` whereas for generators we use ````VoltageRegulatorOn````. The regulating terminal is needed of course, but you did not put it in the documentation page (we have it for generators). Why ? 
Generators: https://www.powsybl.org/docs/iidm/model/generator.html
Shunts: https://github.com/powsybl/powsybl.github.io/pull/114/files

Reply:
> Just a question: regulations in IIDM have a different modeling depending on the object. Here you have chosen Regulating whereas for generators we use VoltageRegulatorOn.

Contrary to shunts which can only be regulated in voltage or off regulation, generators are always regulating and can be regulated in voltage or in reactive power so an attribute `regulating` for generators does not really make sense.

> The regulating terminal is needed of course, but you did not put it in the documentation page (we have it for generators). Why ?

I 've forgotten it, I will add it to the documentation.

Reply:
I am not sure to well understand your answer. Generators can be equiped of automatic generation control and of automatic voltage regulator. The first one is not modeled in IIDM. The second one is a voltage regulation (the first one) and a generator can be in or off voltage regulation.

Reply:
In IIDM, when a generator is off voltage regulation, it is required to be on reactive power regulation. It can not be entirely out of any regulation (in this model). This is why it does not really make sense to have a `regulating` attribute. Shunts, however, can only be on voltage regulation or off regulation altogether. 
Note that I would personally prefer having an enum of `RegulationMode` for generators rather than a `voltageRegulatorOn` attribute, it would be clearer in my opinion...

Reply:
@MioRtia I think I agree with @annetill. For me, it makes the API more complicated to understand if the name of the attributes/method used for the voltage regulation are different for 2 equipments.

Reply:
I agree but it is already the case, voltage regulation for ratio tap changers is indicated by an attribute called `regulating`. However, if you think having `voltageRegulatorOn` as attribute is clearer, I will change it.

Reply:
I still think that having a boolean attribute for generators set as false to indicate that the generator has reactive power regulation is not very user-friendly and I would rather change that. What do you think?

Reply:
Yes, same comments for ratio tap changers. As I said, maybe we have to take some minutes to think about the regulations modeling in IIDM. I am talking about the voltage regulation on a local point in the network and on active power regulation (or current, we need to precise) for phase tap changers. And what about Frequency control ? And what the rest ?

Reply:
I replaced `regulating` by `voltageRegulatorOn` which is indeed clearer about the kind of regulation that is taking place. The modeling of regulation in general is a broader question that will take place, I made an issue about it (cf #1145 )